packages/core-forger/src/manager.ts

```diff
@@ -170,11 +170,13 @@ export class ForgerManager {
                 this.client.emitEvent(ApplicationEvents.TransactionForged, transaction);
             }
         } else {
-            this.logger.warn(
-                `Failed to forge new block by delegate ${this.usernames[delegate.publicKey]} (${
-                    delegate.publicKey
-                }, because there were less than ${minimumMs}ms left`,
-            );
+            if (currentSlot !== roundSlot) {
+                this.logger.warn(`Failed to forge new block by delegate ${prettyName}, because already in next slot.`);
+            } else {
+                this.logger.warn(
+                    `Failed to forge new block by delegate ${prettyName}, because there were less than ${minimumMs}ms left`,
```

Comment:
May also be useful to print the time left and also print time left from what:
```
..., because there were ${timeLeftInMs}ms left in the current slot (less than ${minimumMs}ms)
```